ArkKit/Ark.swift

```diff
@@ -84,7 +100,7 @@ private extension ArkEvent {
         guard let castedAction = action as? any Action<Self> else {
             return
         }
-        
+
```

Comment:
Also, i think `private extension` throws an error in build from our swiftlint:

`No Extension Access Modifier Violation: Prefer not to use extension access modifiers (no_extension_access_modifier)`

Reply:
yikes, ok then just remove the private